### ISSUE TITLE
Save output format

### ISSUE THREAD

**Issue by GregorySchwartz** [DougBurke/hvega#13]
Like altair (https://altair-viz.github.io/user_guide/saving_charts.html), can `hvega` devise a way to save as a `svg`, `pdf`, and `png`?

**Comment by DougBurke** [comment]
I have been relying on IHaskell to do this conversion for me (much improved with Jupyter Lab rather than Jupyter notebook [1]), but I do agree we need functions to

- save the JSON schema
- create other representations (which is what this ticket is about)

From the Altair documentation linked to:

> However, saving these images requires some additional dependencies to run the javascript code necessary to interpret the Vega-Lite specification and output it in the form of an image.

which means this needs thinking about, and it's not likely to be a priority for me, so any input from the community would be lovely.

[1] when run in Jupyter Lab we get interactive view as well as PNG for offline viewing, as shown in  https://nbviewer.jupyter.org/github/DougBurke/hvega/blob/master/notebooks/VegaLiteGallery.ipynb

**Comment by GregorySchwartz** [comment]
I think a good first start is saving as `json` and `html`, following the specifications in altair, as they have an example of the outputted `html`.

**Comment by GregorySchwartz** [comment]
Also, I think it would be neat to save to `html` with options to choose the version, as well as adding custom strings for additional specifications. For instance, you could specify the fictional version 8 with the new supershape mark, without needed to rely on this library being immediately updated each time.

**Comment by DougBurke** [comment]
It depends what version you mean here (as to what the best approach is). I assume you are talking about the version numbers of the JS libraries?

(sorry I'm slow to respond, working on some $DAY_JOB issues)

**Comment by GregorySchwartz** [comment]
Yeah, like the version of `vega` and the like. And, for instance, if there's a new mark available in a new version it would be cool to have a generic mark that's just text or a `Value` for a custom addition.
